Cargo.toml

```diff
@@ -65,3 +73,8 @@ version = "0.3.11"
 
 [features]
 valuable = ["dep:valuable", "valuable-serde", "http", "url"]
+
+otel = [
```

Comment:
Changed to `opentelemetry`. That said, I realised that, strictly speaking, we are integration with `tracing_opentelemetry` rather than `opentelemetry` (at least not directly).
Not sure whether `tracing_opentelemetry` or `opentelemetry` would be a better feature flag.
WDYT?

Reply:
Including the `tracing_` prefix seems redundant,  I think.